addons/website_sale/views/templates.xml

```diff
@@ -2203,60 +2202,54 @@
-                                t-att-title="line._get_shop_warning()"
-                                role="img"
-                                aria-label="Warning"
-                            />
-                        </a>
-                    </t>
-                    <a
-                        t-else=""
-                        href="#"
-                        class="js_add_cart_json d-inline-block btn btn-link border-start-0"
-                        aria-label="Add one"
-                        title="Add one"
-                    >
-                        <i class="fa fa-plus position-relative z-1"/>
+            <t
+                t-set="should_show_quantity_selector"
+                t-value="is_view_active('website_sale.product_quantity')"
+            />
+            <t t-if="should_show_quantity_selector and line._is_sellable()">
+                <a
+                    href="#"
+                    class="js_add_cart_json btn btn-link d-inline-block border-end-0"
+                    aria-label="Remove one"
+                    title="Remove one"
+                >
+                    <i class="position-relative z-1 fa fa-minus"/>
+                </a>
+                <input
+                    type="text"
+                    class="js_quantity quantity form-control border-start-0 border-end-0"
+                    t-att-data-line-id="line.id"
+                    t-att-data-product-id="line.product_id.id"
+                    t-att-value="line._get_displayed_quantity()"
+                />
+                <t t-if="line._get_shop_warning(clear=False)">
+                    <a href="#" class="btn btn-link">
+                        <i
+                            class="fa fa-warning text-warning"
+                            t-att-title="line._get_shop_warning()"
+                            role="img"
+                            aria-label="Warning"
+                        />
                     </a>
                 </t>
-                <t t-else="">
-                    <input
-                        type="hidden"
-                        class="js_quantity form-control quantity"
-                        t-att-data-line-id="line.id"
-                        t-att-data-product-id="line.product_id.id"
-                        t-att-value="line._get_displayed_quantity()"
-                    />
-                </t>
+                <a
+                    t-else=""
+                    href="#"
+                    class="js_add_cart_json d-inline-block btn btn-link border-start-0"
+                    aria-label="Add one"
+                    title="Add one"
+                >
+                    <i class="fa fa-plus position-relative z-1"/>
+                </a>
             </t>
             <t t-else="">
                 <input
-                    type="hidden"
-                    class="js_quantity quantity form-control"
+                    t-att-type="'text'"
```

Comment:
why t-att here? :thinking: 
could be `type="text"` no?

Reply:
Indeed it's a relica of a dynamic implementation where the input was hidden, I'll fix it.